webapp/channels/webpack.config.js

```diff
@@ -384,7 +384,7 @@ async function initializeModuleFederation() {
 
     // Desktop specific code for remote module loading
     moduleFederationPluginOptions.exposes = {
-        './app': 'components/app.jsx',
+        './app': 'components/app.tsx',
```

Comment:
we dont need to specify `tsx` extension here

Reply:
That's right, thanks